src/test/java/persistence/sql/dml/DmlQueryBuilderTest.java

```diff
@@ -0,0 +1,55 @@
+package persistence.sql.dml;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.sql.ddl.H2Dialect;
+import persistence.sql.ddl.Person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class DmlQueryBuilderTest {
+    @DisplayName("클래스 정보를 바탕으로 INSERT 쿼리를 생성한다.")
+    @Test
+    void insert() {
+        final DmlQueryBuilder dmlQueryBuilder = new DmlQueryBuilder(new H2Dialect());
+        final Person person = new Person(1L, "Kent Beck", 64, "beck@example.com");
+        assertEquals(expectedForInsert(), dmlQueryBuilder.insert(Person.class, person));
```

Comment:
저는 테스트 기댓값을 테스트와 최대한 가까이 작성하는 것이 가독성에 더욱 좋다고 생각하는데요!
주원님은 어떤 기준으로 분리해주셨는지 궁금합니다 😄 

```suggestion
        String insert = dmlQueryBuilder.insert(Person.class, person);
        String expected = "INSERT INTO USERS (id, nick_name, old, email) VALUES (1, 'Kent Beck', 64, 'beck@example.com');";
```

Reply:
말씀해주신 부분이 잘 이해가 가지 않아서요 ㅜㅜ 다른 방법으로 설명해주실 수 있을까요?!

Reply:
작성해주신 테스트에서 검증 부분의 `expectedForInsert()` 메서드를 구현해주셨는데요!

메서드 분리 대신 문자열을 그대로 테스트에 함께 작성하는 것이 더욱 가독성이 좋지 않을까 하여서 드렸던 부분입니다! 😄